### DISCUSSION TITLE
第25期：「idea分享」浏览器里的图片批处理工具构想 | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#66] [General]
# 第25期：「idea分享」浏览器里的图片批处理工具构想 | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-11-11_025

**Comment by huyixi** [comment]
Image Downloader 这个浏览器插件可以实现你期待的一些功能：https://chromewebstore.google.com/detail/image-downloader/cnpniohnfphhjihaiiggeabnkjhpaldj

**Comment by taingh** [comment]
可以看一下这个插件 如果仍然不能解决你的痛点的话 就证明仍然还有需求没被满足 我可以考虑尝试实现一个新的

https://chromewebstore.google.com/detail/%E6%89%B9%E9%87%8F%E5%9B%BE%E7%89%87%E4%B8%8B%E8%BD%BD%E5%99%A8-imageye/agionbommeaifngbhincahgmoflcikhm
